hail/www/index.md

```diff
@@ -21,7 +21,7 @@ Contact the Hail team at <a href="mailto:hail@broadinstitute.org"><code>hail@bro
 
 ### Citing Hail
 
-If you use Hail for published work, please cite the software. You can get a
+If you use Hail for published work, please cite the software. We maintain a [list of publications and projects](references.md) that have used Hail and would like to know about your work. You can get a
```

Comment:
Suggested rewrite:
If you use Hail for published work, please cite the software and add a link to your work to our growing [list of Hail-powered science](references.md).